### PR TITLE
TE-14.1: Adding changes from ATE #1067 to OTG tests

### PR DESCRIPTION
**Pull request by stthummuri**
Porting changes from #1067 and #1068 to OTG location

1. For dutPort2 and atePort2, unit 0 subinterface is configured without vlan(vlan-id 0). Added deviation to configure vlan for all subinterfaces to avoid mix of tagged and untagged subinterfaces.

2. Added network instance type for DEFAULT instance

3. Index 1-64 are used for programming NH which is used for prefixes in default instance. Updated NH index start value for vrf1(nhID1) from 65, vrf2(nhID2) from 1065 and vrf3(nhID3) from 18565

### CODE REVIEW

**Comment by LimeHat** [comment]
Did you use an old branch as a base?

Looks like this PR is undoing changes from #1236 and #1235 in addition to what is stated.

File: feature/gribi/otg_tests/gribi_scaling/gribi_scaling_test.go [hunk 1]
```diff
@@ -347,7 +345,13 @@ func generateSubIntfPair(t *testing.T, top gosnappi.Config, dut *ondatra.DUTDevi
 	nextHops := []string{}
 	nextHopCount := 63 // nextHopCount specifies number of nextHop IPs needed.
 	for i := 0; i <= nextHopCount; i++ {
+		//yang file using  vlan rang 1-4094 https://github.com/openconfig/public/blob/b34db05e8cf2efe69df3762d4bbd80665e1f9e79/release/models/vlan/openconfig-vlan-types.yang#L133
```
Comment (greg-dennis) [inline feature/gribi/otg_tests/gribi_scaling/gribi_scaling_test.go:348]: "//yang" -> "// Yang"

**Review by greg-dennis — DISMISSED** [review]
One nit

File: feature/gribi/otg_tests/gribi_scaling/gribi_scaling_test.go [hunk 2]
```diff
@@ -301,13 +301,11 @@ func createVrf(t *testing.T, dut *ondatra.DUTDevice, d *oc.Root, vrfs []string)
 		if vrf != *deviations.DefaultNetworkInstance {
 			i := d.GetOrCreateNetworkInstance(vrf)
 			i.Type = oc.NetworkInstanceTypes_NETWORK_INSTANCE_TYPE_L3VRF
+			i.GetOrCreateProtocol(oc.PolicyTypes_INSTALL_PROTOCOL_TYPE_STATIC, *deviations.StaticProtocolName)
```
Comment (LimeHat) [inline feature/gribi/otg_tests/gribi_scaling/gribi_scaling_test.go:304]: Please revert this change or update the PR description

File: feature/gribi/otg_tests/gribi_scaling/gribi_scaling_test.go [hunk 3]
```diff
@@ -301,13 +301,11 @@ func createVrf(t *testing.T, dut *ondatra.DUTDevice, d *oc.Root, vrfs []string)
...
 			gnmi.Replace(t, dut, gnmi.OC().NetworkInstance(vrf).Config(), i)
 			nip := gnmi.OC().NetworkInstance(vrf)
 			fptest.LogQuery(t, "nonDefaultNI", nip.Config(), gnmi.GetConfig(t, dut, nip.Config()))
 		}
-		if *deviations.ExplicitGRIBIUnderNetworkInstance {
```
Comment (LimeHat) [inline feature/gribi/otg_tests/gribi_scaling/gribi_scaling_test.go:308]: Please revert this change or update the PR description

File: feature/gribi/otg_tests/gribi_scaling/gribi_scaling_test.go (hunk 1 above)
Comment (stthummuri) [reply feature/gribi/otg_tests/gribi_scaling/gribi_scaling_test.go]: Done

File: feature/gribi/otg_tests/gribi_scaling/gribi_scaling_test.go (hunk 2 above)
Comment (stthummuri) [reply feature/gribi/otg_tests/gribi_scaling/gribi_scaling_test.go]: Done

[the remark this answers is not in the document]
Comment (stthummuri) [reply feature/gribi/otg_tests/gribi_scaling/gribi_scaling_test.go]: Added back the *deviations.ExplicitGRIBIUnderNetworkInstance

**Review by greg-dennis — APPROVED** [review]
Since my last approval was dismissed, re-approving for Go quality only
